MAUI/Shimmer/Customization.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the Customization of Syncfusion .NET MAUI Shimmer (SfShimmer) control, its basic features, elements and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Customization of .NET MAUI Shimmer (SfShimmer)
+
+The `SfShimmer` control provides the following properties to customize the Shimmer,
+
+* `Fill` - To customize the background color of Shimmer view. 
+* `WaveColor` - To customize the Shimmer wave color.
+* `WaveWidth` - To customize the Shimmer wave width.
+* `WaveDirection` - To customize the Shimmer wave direction.
+* `RepeatCount` - To customize the repeat count for the Shimmer.
+* `AnimationDuration` - To customize the animation duration of the wave.
+
+This section explains how to customize the Shimmer control.
+
+## Fill
+
+The background color of the `SfShimmer` view can be customized by using the `Fill` property. The default value of the `Fill` property is `#F3EDF7`.
```

Comment:
Highlight the property lines for all the topics

Reply:
Highlighted the property lines for all the topics